ui/src/components/JsonDisplay/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { string, object } from 'prop-types';
+import { dump } from 'js-yaml';
+import { Switch, Typography, Grid } from '@material-ui/core';
+import Code from '../Code';
+import 'highlight.js/styles/atom-one-dark.css';
+
+const toYaml = obj => {
+  return dump(obj);
+};
+
+const toJson = obj => {
+  return JSON.stringify(obj, null, 2);
+};
+
+const YamlJsonSwitch = ({ handleChange, value }) => {
```

Comment:
I think it'd be cleaner and easier to read if you moved this whole component into the `JsonDisplay` component and then move the two helper functions (toYaml and toJson) inline since they only look to be used once.